tests/components/cambridge_audio/fixtures/get_presets_list.json

```diff
@@ -28,7 +28,7 @@
       "name": "Unknown Preset Type",
       "type": "Unknown",
       "class": "stream.unknown",
-      "state": "OK"
+      "state": "UNAVAILABLE"
```

Comment:
These fixture changes (OK -> UNAVAILABLE) feel off, is it done on purpose?

Reply:
That being state "OK" was a mistake, but wasn't relevant until now. Upon further testing with the device, "UNAVAILABLE" would be the proper state here.